clients/js/src/revisions/v1/rule.ts

```diff
@@ -1,5 +1,18 @@
-import { AmountOperator } from '../shared';
-import { PublicKeyAsArrayOfBytes } from './publicKey';
+import type { RuleV2 } from '../v2';
+import { AdditionalSignerRuleV1 } from './additionalSigner';
+import { AllRuleV1 } from './all';
+import { AmountRuleV1 } from './amount';
+import { AnyRuleV1 } from './any';
+import { NamespaceRuleV1 } from './namespace';
+import { NotRuleV1 } from './not';
+import { PassRuleV1 } from './pass';
+import { PdaMatchRuleV1 } from './pdaMatch';
+import { ProgramOwnedRuleV1 } from './programOwned';
+import { ProgramOwnedListRuleV1 } from './programOwnedList';
+import { ProgramOwnedTreeRuleV1 } from './programOwnedTree';
+import { PubkeyListMatchRuleV1 } from './pubkeyListMatch';
+import { PubkeyMatchRuleV1 } from './pubkeyMatch';
+import { PubkeyTreeMatchRuleV1 } from './pubkeyTreeMatch';
```

Comment:
Could we make all these `import type` to avoid circular dependencies of the sort:

`v1/rule -> v1/additionalSignerRuleV1 -> v1/rule`